src/components/EditableField.jsx

```diff
@@ -96,6 +96,7 @@ const EditableField = (props) => {
       return (
         <>
           <Input
+            data-testid='editableFieldInput'
```

Comment:
What is this used for? Is it for the integration tests?

Reply:
no, for the unit tests, Im using `getByTestId`  from rtl